src/hotspot/cpu/x86/stubGenerator_x86_64_poly.cpp

```diff
@@ -559,7 +558,6 @@ void StubGenerator::poly1305_process_blocks_avx512(
   const Register t0 = r13;
   const Register t1 = r14;
   const Register t2 = r15;
-  const Register rscratch = r13;
```

Comment:
After seeing the whole patch, I noticed that `t1` is used only at the very end of the stub. 

Alternatively, you could keep `rscratch` and move `t1` declaration close to the use sites.

Reply:
No (unless I misunderstood), there are a couple more in the middle passed to `poly1305_multiply_scalar` (I prefer 'the look' of `t0, t1, t2`)

```
  poly1305_multiply_scalar(a0, a1, a2,
                           r0, r1, c1, true,
                           t0, t1, t2, mulql, mulqh);
```

Reply:
Yes, you are right. Overlooked those when browsed code on GitHub.